tests/testthat/test-derive_merged.R

```diff
@@ -348,10 +348,53 @@ test_that("derive_vars_merged Test 15: error if not unique, no order, check_type
   )
 })
 
+## Test 16: merge relationship as 'many-to-one' ----
+test_that("derive_vars_merged_summary Test 16: merge relationship as 'many-to-one'", {
+  actual <- derive_vars_merged(advs,
+    dataset_add = adsl,
+    by_vars = exprs(USUBJID),
+    new_vars = exprs(SEX),
+    relationship = "many-to-one"
+  )
+
+  expected <- left_join(advs, select(adsl, USUBJID, SEX), by = "USUBJID")
+
+  expect_dfs_equal(
+    base = expected,
+    compare = actual,
+    keys = c("USUBJID", "AVISIT")
+  )
+})
+
+## Test 17: error incorrect 'one-to-one' ----
+test_that("derive_vars_merged_summary Test 17: error incorrect 'one-to-one'", {
+  expect_snapshot(
+    derive_vars_merged(advs,
+      dataset_add = adsl,
+      by_vars = exprs(USUBJID),
+      new_vars = exprs(SEX),
+      relationship = "one-to-one"
+    ),
+    error = TRUE
+  )
+})
+
+## Test 18: merge sel vars 'one-to-one' ----
+test_that("derive_vars_merged_summary Test 18: merge sel vars 'one-to-one'", {
```

Comment:
Please update the function name in the title. (This should be done automatically if the addin is called.)